src/Command/Dedupe.php

```diff
@@ -20,14 +20,18 @@ protected function configure(): void
     {
         $this
             ->setDescription('Removes duplicated messages')
-            ->addArgument('field', InputArgument::REQUIRED, 'Name of the field which identifies each message')
+            ->addArgument('field', InputArgument::REQUIRED | InputArgument::IS_ARRAY, 'Name of the field which identifies each message')
         ;
     }
 
     protected function execute(InputInterface $input, OutputInterface $output): void
     {
-        $field = $input->getArgument('field');
-        \assert(\is_string($field));
+        /** @var string[] $fields */
+        $fields = $input->getArgument('field');
+        \assert(\is_array($fields));
+
+        // Keep field names sorted to help the hash building
+        sort($fields);
```

Comment:
does the sorting matter? the order is stable if we use the order from the argument and not the order inside the message.

Reply:
Agree, I just wanted to be extra sure about that sorting :) code removed